v2/yarpcthrift/thriftrw-plugin-yarpc2/roundtrip_test.go

```diff
@@ -195,7 +200,13 @@ func testRoundTrip(t *testing.T, enveloped, multiplexed bool) {
 			newClientFunc: storeclient.New,
 			method:        "Integer",
 			methodArgs:    []interface{}{ptr.String("foo")},
-			wantError:     &atomic.KeyDoesNotExist{Key: ptr.String("foo")},
+			// TODO(mhp): After we have a way to map errors between YARPC errors
+			// and thrift exceptions, this should be revisited so that the check
+			// below actually returns well-defined thrift exceptions.
+			wantError: yarpcerror.WrapHandlerError(
+				&atomic.KeyDoesNotExist{Key: ptr.String("foo")},
+				"roundtrip-server",
+				"ReadOnlyStore::integer"),
```

Comment:
nit: We can use an additional trailing `,` and wrap the `)` around so that it's in line with the `yarpcerror.WrapHandlerError` declaration.